app/assets/stylesheets/_sponsor_application.scss

```diff
@@ -78,3 +78,75 @@
 .steps.active:after {
   background: rgba(0, 0, 0, 0.5);
 }
+
+.status{
+  margin: 20px 0;
+  clear: both;
+}
+
+.status__heading{
+  line-height: 25px;
+  margin: 0;
+  margin-left: 10px;
+  float: left;
+  margin-bottom: 5px;
+  font-weight: normal;
+}
+
+.status__body{
+  clear: both;
+  margin-left: 35px;
+  max-width: 600px;
+}
+
+.status__icon{
+  float: left;
+  padding: 5px;
+  border-radius: 50px;
+  overflow: hidden;
+}
+
+.status__actions{
+  margin-left: 35px;
+  margin-top: 20px;
+}
+
+.status__actions button{
+  margin-right: 50px;
+}
+
+.status__icon:after{
+  width: 15px;
```

Comment:
Properties should be ordered background-position, background-repeat, background-size, content, display, height, width